torch/distributed/_shard/sharded_tensor/utils.py

```diff
@@ -161,6 +174,19 @@ def build_metadata_from_local_shards(
 
 
 def build_global_metadata(gathered_metadatas: Sequence[Optional[ShardedTensorMetadata]]):
```

Comment:
Please remove the new dosctrings and only fix the errors listed in the issue. I think the issues clearly asks to skip the "Missing docstring" errors.